siliconcompiler/utils.py

```diff
@@ -132,6 +134,8 @@ def get_default_iomap():
     default_iomap.update({ext: ('constraint', 'pcf') for ext in constraint_pcf})
 
     default_iomap.update({ext: ('fpga', 'xdc') for ext in fpga_xdc})
+    default_iomap.update({ext: ('layout', 'place') for ext in fpga_place})
+    default_iomap.update({ext: ('layout', 'route') for ext in fpga_route})
```

Comment:
I think these should go into fpga/vpr_place and fpga/vpr_route to make it clear these are VPR files